<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDlg</class>
 <widget class="QDialog" name="LoginDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="LoginDlg_btLogin">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LoginDlg_btQuit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>250</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>关机</string>
   </property>
  </widget>
  <widget class="QComboBox" name="LoginDlg_cbName">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="LoginDlg_txtPassword">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="LoginDlg_lbBG">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LoginDlg_lbLogo">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LoginDlg_lbMessage">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>LoginDlg_lbBG</zorder>
  <zorder>LoginDlg_btLogin</zorder>
  <zorder>LoginDlg_btQuit</zorder>
  <zorder>LoginDlg_cbName</zorder>
  <zorder>LoginDlg_txtPassword</zorder>
  <zorder>LoginDlg_lbLogo</zorder>
  <zorder>LoginDlg_lbMessage</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
